pkg/model/firewall.go

```diff
@@ -59,6 +75,14 @@ func (b *FirewallModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	// b.applyNodeToMasterAllowSpecificPorts(c)
 	b.applyNodeToMasterBlockSpecificPorts(c, nodeGroups, masterGroups)
 
+	for _, task := range b.SecurityGroupRules {
+		finalRules[fi.StringValue(task.Name)] = task
```

Comment:
It looks like the intention here is to replace an existing rule that might already be in the map per the call to `(*FirewallModelBuilder).getExistingRulesFromCloud` above.

Reply:
Correct